src/main/java/Main.java

```diff
@@ -0,0 +1,12 @@
+public class Main {
+    public static void main(String[] args) {
+        int countFigure = (int) (Math.random() * (10 + 1) + 1);
```

Comment:
можем вынести число 11 как константу
```suggestion
        int countFigure = (int) (Math.random() * (11) + 1);
```